### PR TITLE
AV-75535 CSS abstract + 3rd level TOC

### PR DESCRIPTION
**Pull request by osfameron**
Requested by Richard

### CODE REVIEW

**Review by RichardSmedley — APPROVED** [review]
LGTM (the same changes worked on staging).

Thanks @osfameron  :+1:

File: src/css/doc.css [hunk 1]
```diff
@@ -499,6 +499,8 @@ ul ul ul {
 
 .doc .abstract blockquote {
   margin: 0;
+  padding-left: 0.5em;
```
Comment (sarahlwelton) [inline src/css/doc.css]: Thoughts on tweaking this to `1.5em` so we get something more like: 

image

**Review by sarahlwelton — COMMENTED** [review]
Small thing

File: src/css/doc.css (hunk 1 above)
Comment (osfameron) [reply src/css/doc.css]: That looks fine to me 👍

Comment (sarahlwelton) [reply src/css/doc.css]:
```suggestion
  padding-left: 1.5em;
```

**Pull request merged**